backend/src/test/java/ai/verta/modeldb/PublicTestSequenceSuite.java

```diff
@@ -22,6 +27,11 @@
   RepositoryTest.class,
   CommitTest.class,
   DiffTest.class,
-  MetadataTest.class
+  MetadataTest.class,
```

Comment:
Please verify these tests work without uacservice

Reply:
All 5 tests are unit tests, they should work without dependencies and without uac